Yoast/wpseo-news
# Use right publish date format

## Summary

This PR can be summarized in the following changelog entry:
- [Bugfix] Fixes a bug where the wrong timezone has been used in the sitemap

## Relevant technical choices:

* Copied the logic from Yoast SEO to fallback by guessing the timezone

## Test instructions

This PR can be tested by following these steps:

* Add a post and see if the timezone will be correct.

Fixes #227
